content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+
+By developing such an ontology to connect disciplines, Sperber proposes a credible bridge between the notions of representation in social science and that of mental representation in cognitive science, without reducing one area to the other or making simplistic assumptions about the phenomena encountered.
+In this sense, his proposal is that of a naturalistic ontology for the study of culture which builds on cognitive science principles.
+It is amenable to anthropology, and encourages the combination of the two bodies of knowledge in a well defined way.
+Sperber can then rephrase interdisciplinary questions in terms of spread and transformation of representations.
+For instance:
+what types of representations are less transformed than others as people integrate them or perceive them, and produce them anew, making them circulate in a society?
+Such representations, spreading wider than others, become *cultural representations* and will characterise a given society (for instance the habit of clothing, eating customs or values, or technological knowledge).
+
+Why are those representations so stable, and how do they evolve?
+Sperber introduces an additional concept to analyse this evolution:
+the dynamical system of representations which models a society's culture exhibits attractors, called *cultural attractors*, that depend on the complex interaction of psychological and ecological factors, and on the distribution of representations at a given moment in time [@sperber_explaining_1996, p. 106-118].
+Cultural attractors are one of the core concepts in CAT providing intelligibility to the evolution of culture and to the reciprocal influence of psychology, culture, and environment.
+As such, a central goal in the CAT research stream has been to identify existing attractors, and explain their emergence based on the interaction of psychological and ecological factors.
+
+The most important intuition in Sperber's proposal, and what differentiates it from previous works, is the centrality of psychology for the evolution of culture [@sperber_explaining_1996, p. 31] and its role in the emergence of attractors.
+He substantiates this by relying heavily on contemporary cognitive science, and in particular by adopting and extending the view of the modularity of mind initially defended by @fodor_modularity_1983 (his view goes further than Fodor's, as he argues for a *massive modularity of mind* applying not only to perception but also to conceptual processes).
+Combining this view with epidemiology of representations, Sperber argues, results in a theory avoiding both the blank slate approach to psychology [@sperber_explaining_1996, p. 63-66] and a naive application of neo-darwinist formalism to the specific case of culture [@sperber_explaining_1996, p. 101], while still being able to account for the diversity of cultures [@sperber_explaining_1996, p. 120].
```

Comment:
### Camille
"blank slate approach to psychology", qu'est-ce donc ?  pareil: "a naive application of neo-darwinist formalism to the specific case of culture", on doit comprendre sans devoir retourner vers Sperber 63-66 ou Sperber 101 (I, for one, left it in Berlin: see...).  Talking of which, dans cette première description des différences entre SCE et CAT, je ne suis plus trop raccord sur le claim central de SCE.  Peut-être développer 1.1.2.1 du coup.

Reply:
Done.